src/universal/utils/transit.js

```diff
@@ -57,4 +57,32 @@ export default transit.withExtraHandlers([
       : require('url').parse(value)
     ),
   },
-]);
+];
+
+const modifiedHandlers = handlers.withExtraHandlers(extraHandlers);
+
+const reader = transit.reader('json', {
+  handlers: modifiedHandlers.read,
+  cache: false,
+  // Copied from transit-immutable-js. Without this, Error fails to read.
```

Comment:
note: license is MIT
* https://github.com/glenjamin/transit-immutable-js/blob/v0.8.0/package.json#L18
* https://github.com/glenjamin/transit-immutable-js/blob/v0.8.0/LICENSE